src/Checkout/Mixin.js

```diff
@@ -499,13 +499,16 @@ export default {
                     return
                 }
 
-                if (response.data.paymentResponse.status === 'succeeded') {
+                /**
+                 * Check for Stripe or PayPal success statuses
+                 */
+                if (response.data.paymentResponse.status === 'succeeded' ||
```

Comment:
Might be nice to have a method similar to the PHP version `hasSuccessfulPayment` to clean this up a little bit?